test/functional/cmdLineTests/criu/playlist.xml

```diff
@@ -133,6 +133,41 @@
 			<impl>openj9</impl>
 		</impls>
 	</test>
+	<test>
+		<testCaseName>cmdLineTester_criu_nonPortableRestore_jdk_crac</testCaseName>
+		<variations>
+			<variation>-Xgcpolicy:optthruput</variation>
+			<variation>-Xgcpolicy:optavgpause</variation>
+			<variation>-Xgcpolicy:gencon -Xgcthreads64 -XX:CheckpointGCThreads=1</variation>
+			<variation>-Xgcpolicy:gencon -Xgcthreads1</variation>
+		</variations>
+		<disables>
+			<disable>
+				<comment>https://github.com/eclipse-openj9/openj9/issues/18468</comment>
+				<platform>ppc64le.*</platform>
+			</disable>
+		</disables>
+		<command>
+			$(JAVA_COMMAND) $(CMDLINETESTER_JVM_OPTIONS) -Xdump \
+			-DSCRIPPATH=$(TEST_RESROOT)$(D)criuScript.sh -DTEST_RESROOT=$(TEST_RESROOT) \
+			-DJAVA_COMMAND=$(JAVA_COMMAND) -DJVM_OPTIONS=$(Q)$(JVM_OPTIONS)$(Q) \
+			-jar $(CMDLINETESTER_JAR) -config $(Q)$(TEST_RESROOT)$(D)criu_nonPortable_jdk_crac.xml$(Q) \
+			-explainExcludes -xids all,$(PLATFORM),$(VARIATION) -nonZeroExitWhenError; \
+			$(TEST_STATUS)
+		</command>
+		<features>
+			<feature>CRIU:required</feature>
```

Comment:
we will need another feature flag for crac

Reply:
Added a feature `CRAC`, currently specified along with `CRIU` when `org.eclipse.openj9.criu.isCRIUCapable` is detected.
It could be set separately when `CRAC` has its use scenario.